procedures/release-duty/merge-duty/merge_duty.rst

```diff
@@ -174,10 +174,17 @@ Merge beta to release
 1. Upon successful run, ``mozilla-release`` should get a version bump
    and branding changes consisting of a ``commit`` like
    `this <https://hg.mozilla.org/releases/mozilla-release/rev/118aa10ac456d05606f113ade5c26ae4637081ce>`__
-   and a ``tag`` like
-   `this <https://hg.mozilla.org/releases/mozilla-release/rev/80431bd8e840dc52327d3c881e5e679f1002b0da>`__
-2. In the same time ``mozilla-beta`` should get a tag like
-   `this <https://hg.mozilla.org/releases/mozilla-beta/rev/f0415f3c8bc959f2ac8c6ded6f9c501a49a97e11>`__
+   and two new tags. The first tag should be
+   `in the form FIREFOX_RELEASE_xxx_END <https://hg.mozilla.org/releases/mozilla-release/rev/92e4f64aabfb73736f7e2486802d8deb54dbf111>`__
+   - where the xxx is the major Gecko version that Release had prior to the merge. The other tag
+   should be
+   `in the form FIREFOX_RELEASE_yyy_BASE <https://hg.mozilla.org/releases/mozilla-release/rev/c4ed5781ba9260b6a46b97be4c66f32a28eea1a6>`__
+   - where the yyy is the major Gecko version that Release now has.
+
+2. In the same time ``mozilla-beta`` should get a tag
```

Comment:
*At* the same time might be better (nit)

Reply:
Good catch - thanks!